### DISCUSSION TITLE
Outdoors map style preview ???

### DISCUSSION THREAD

**Discussion by steveparry1** [organicmaps/organicmaps#6620] [Q&A] [answered]
Today on the Google Play Store under 'What's new' it states "Outdoors map style preview - read on how to activate it on maps.org/news" ...  I can find no info on how to do this on the website. Can anyone enlighten me? Thanks.

**Comment by hb0nd** [comment, marked as answer]
type `?olight` into the search bar for outdoor light mode, and `?odark` for outdoor dark mode.
`?light` will return to the default light style

**Reply by wysocki** [reply]
When I type those strings in the search bar, it just returns a list of places that match (like "Oliphant Park")
